app/models/event_stream.rb

```diff
@@ -32,4 +32,48 @@ def emit_notifications
   rescue => err
     _log.log_backtrace(err)
   end
+
+  #
+  # Purging methods
+  #
+
+  def self.keep_ems_events
+    VMDB::Config.new("vmdb").config.fetch_path(:ems_events, :history, :keep_ems_events)
```

Comment:
I realize this is a move, but the names of these methods and this path seem bad now.   I wonder if we should fix them in a followup PR.